packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -53,6 +53,7 @@ export type SubscriptionCreateProps = {
   customer: Customer | null;
   selectedPlan: Plan;
   refreshSubscriptions: () => void;
+  setTransactionInProgress: (value: boolean) => void;
```

Comment:
Wouldn't we want to use a hook for this; i.e. `useState` similar to `setPaypalScriptLoaded`? Setting this to `true` in this way on clicking the `PayPalButton` wouldn't trigger a re-render of `SubscriptionCreate`.

Reply:
Edit: My bad you do use `useState`... hmmm... I don't see the screen though when I try it locally.

Reply:
Also why is `setTransactionInProgress` a prop here on `SubscriptionCreate`? I don't see anything other than a noop passed in for the tests.

Reply:
Answering my own questions here...

> Also why is setTransactionInProgress a prop here on SubscriptionCreate? I don't see anything other than a noop passed in for the tests.

Because you set it to `true` when the `PayPalButton` is clicked which is inside `SubscriptionCreate`.

Also I have a theory on why it may not be working, I think we'd want to pass the state to `SubscriptionCreate` and conditionally show `PaymentProcessing` there, since `Product` will only render `SubscriptionSuccess` after the processing is complete, so we'll never see it if it's conditionally shown there.